airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1429,8 +1428,9 @@ class BigQueryCreateExternalTableOperator(GoogleCloudBaseOperator):
     :param table_resource: Table resource as described in documentation:
         https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#Table
         If provided all other parameters are ignored. External schema from object will be resolved.
-    :param schema_object: If set, a GCS object path pointing to a .json file that
-        contains the schema for the table. (templated)
+    :param gcs_schema_object: Full path to the JSON file containing
```

Comment:
we should not rename it directly otherwise this would be a breaking change.

@attilaszombati if really want to make this change, shall we deprecate it first wdyt?

Reply:
Thanks for the review @pankajastro.

I think we should avoid breaking changes, but I'm not sure about the deprecation process. Could you help me with how can we do that, do you have documentation about this?

Reply:
here is example https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/operators/bigquery.py#L1272

Reply:
@pankajastro I managed to find a way to make make it without a breaking change. Could you take a look at it?